docs/bir-spec/src/main/resources/kaitai/bir.ksy

```diff
@@ -467,17 +467,18 @@ types:
         type: record_field
         repeat: expr
         repeat-expr: record_fields_count
-      - id: has_init_function
-        type: s1
-      - id: record_init_function
-        type: record_init_function
-        if: has_init_function == 1
       - id: type_inclusions_count
         type: s4
       - id: type_inclusions_cp_index
         type: s4
         repeat: expr
         repeat-expr: type_inclusions_count
+      - id: default_values
```

Comment:
Shall we update the BIR version?

Reply:
Updated https://github.com/ballerina-platform/ballerina-lang/pull/40579/commits/f0dcb27b28515f053eaf07ad0dfd3032c485fc90